modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -3544,6 +3544,110 @@ RETURN *
 ======
 =====
 
+[#_neo_clientnotification_statement_redundantoptionalprocedure]
+=== Redundant Optional Procedure
+
+.notification details
+[cols="<1s,<4"]
+|===
+|neo4j code
+m|neo.clientnotification.statement.redundantoptionalprocedure
+|title
+a|The use of `OPTIONAL` is redundant when the procedure calls a void procedure.
+|description
+|The use of `OPTIONAL` is redundant as `CALL %s` is a void procedure.
+|category
+m|generic
+|gqlstatus code
+m|03n61
+|status description
+a|info: redundant optional procedure. The use of `OPTIONAL` is redundant as `CALL %s` is a void procedure.
+|classification
+m|generic
+|severitylevel
+m|information
+|===
+
+.Redundant use of optional on a procedure call.
+[.tabbed-example]
+=====
+[.include-with-neo4j-code]
```

Comment:
You must also add a tab example for the GQLSTATUS code, similar to the rest of the notifications.

Reply:
I think I've misunderstood the format. I'm pushing some changes to align with the rest of the notifications. I'll make sure that the notifications in the monorepo correspond to what is written here, in substance and formatting.